.drone.yml

```diff
@@ -1269,6 +1269,7 @@ trigger:
     - '*.md'
     include:
     - pkg/**
+    - public/*.json
```

Comment:
rather than change this (triggering a failure) make a minor change to something like:
https://github.com/grafana/grafana/blob/d5fda0614773d5ed12df53b939eb5e3027df2960/pkg/api/alerting.go#L219

maybe "Get alert by ID." >> "Get alert by internal ID."

Reply:
i'll change something non-exported, so it doesn't break your manual fixes - but yeah good call, an arbitrary change like this is cleaner.